sos/report/plugins/ceph_osd.py

```diff
@@ -103,15 +112,21 @@ def setup(self):
                 "/var/snap/microceph/common/logs/*ceph-osd*.log",
             ])
 
-    def get_socks(self):
+        # common add_cmd_output for ceph and microceph
+        self.add_cmd_output(
+          [f"ceph daemon {i} {c}" for i in self.get_socks(
+               directory) for c in cmds]
```

Comment:
Minor nit - the line-break here is a bit odd at first glance. This, imo, reads a little easier if we break it with the entire call:

```python
self.add_cmd_output([
    f"ceph daemon {i} {c}" for i in
    self.get_socks(directory) for c in cmds
 ])
```

Reply:
ack thanks, done.